src/QsCompiler/Tests.Compiler/TestCases/QirTests/TestOpCall2.ll

```diff
@@ -9,12 +9,12 @@ entry:
   %4 = call i8* @__quantum__rt__array_get_element_ptr_1d(%Array* %3, i64 0)
   %5 = bitcast i8* %4 to %Qubit**
   store %Qubit* %control, %Qubit** %5
-  %__controlQubits____inline__1 = call %Array* @__quantum__rt__array_concatenate(%Array* %__controlQubits__, %Array* %3)
-  call void @__quantum__rt__array_update_alias_count(%Array* %__controlQubits____inline__1, i64 1)
-  call void @__quantum__qis__x__ctl(%Array* %__controlQubits____inline__1, %Qubit* %target)
-  call void @__quantum__rt__array_update_alias_count(%Array* %__controlQubits____inline__1, i64 -1)
+  %__controlQubits__1 = call %Array* @__quantum__rt__array_concatenate(%Array* %__controlQubits__, %Array* %3)
+  call void @__quantum__rt__array_update_alias_count(%Array* %__controlQubits__1, i64 1)
+  call void @__quantum__qis__x__ctl(%Array* %__controlQubits__1, %Qubit* %target)
+  call void @__quantum__rt__array_update_alias_count(%Array* %__controlQubits__1, i64 -1)
   call void @__quantum__rt__array_update_reference_count(%Array* %3, i64 -1)
-  call void @__quantum__rt__array_update_reference_count(%Array* %__controlQubits____inline__1, i64 -1)
+  call void @__quantum__rt__array_update_reference_count(%Array* %__controlQubits__1, i64 -1)
```

Comment:
I'm not sure how easy this is to do, but is it worth having a test that verifies the increment occurs when a second instance of the named inline is needed, ie: `%__controlQubits__2`?

Reply:
It's harder than you would think to come up with a test case due to all the caching. I added it to the inlining test.